scripts/base/init-bare.zeek

```diff
@@ -4276,6 +4276,11 @@ export {
 		dig_product_id:         string &optional;
 	};
 
```

Comment:
Maybe this should be commented, to state what the meaning of the values here are, or reference where that information is

Reply:
If I'm not mistaken you can find information about ClientCoreData [here](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/00f1da4a-ee9c-421a-852f-c19f92343d73) and ClientSecurityData [here](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/6b58e11e-a32b-4903-b736-339f3cfe46ec). It does seem like some comments could be nice.